OpenSourceRecipe/Models/Recipe.cs

```diff
@@ -46,3 +45,35 @@ public class Recipe
     // Foreign Key to the Cuisine table
     public int CuisineId { get; set; }
 }
+
+public class CreateRecipeDto
+{
+    public string? RecipeTitle { get; set; }
+    public string? TagLine { get; set; }
+    public int Difficulty { get; set; }
+    public int TimeToPrepare { get; set; }
+    public string? RecipeMethod { get; set; }
+    public string? RecipeImg { get; set; }
+    public string? Cuisine { get; set; }
+    public int? ForkedFromId { get; set; } = null;
```

Comment:
Tiny thing to note: The `= null` is essentially redundant. Any nullable variable should be null by default when defined.